readme.md

```diff
@@ -1,10 +1,43 @@
 # loggyslav [![Build Status](https://travis-ci.org/degordian/loggyslav.svg?branch=master)](https://travis-ci.org/degordian/loggyslav)
 
-TODO
+### What can loggyslav do for you?
+- Make your `Typescript/Javascript` logging easier
+- Prevent you from writing `console.log` all over the code
+- Force you to write better code by encouraging you to write smaller methods and classes so you can easily log them 
+- Give you a great tool with smart logging which enables you to easily trace and resolve possible bugs
+
+[![Tell me more](https://media.makeameme.org/created/curious-tell-me.jpg)](https://github.com/degordian/loggyslav)
+
+> Loggyslav is an npm module which easily attaches to any `class`, `method` or `property` and watches for any call or change.
+
+> When attached to a `method` it can easily track every time it is called. It can log `input parameters` and `return value`.
+
+> When attached to a `property` it can easily track every time it is updated.
 
-# Usage
+### Usage
 ```shell
 TODO
```

Comment:
This is the most important part :) 
Users need to know how to use it, put a simple example, and a logging example